ocaml/testsuite/tests/typing-layouts-float64/basics_alpha.ml

```diff
@@ -2,7 +2,6 @@
    flags = "-extension layouts_alpha"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are unreviewed *)
```

Comment:
Line 57:  done
Line 284: done
Line 362: done
Line 571: filed ticket for this
Line 625: done
Line 636: filed ticket. this is easier to fix after merging back to main because a corner case around `Pcl_let`
Line 639: done
Several: rephrased all messages to refer to types
Line 660: same as 636
Line 752: we implemented something for type arguments of imported types, but not for imported functions. filed a ticket for this